src/main/webapp/video_context.js

```diff
@@ -12,30 +12,30 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-google.charts.load('current', {'packages': ['corechart']});
-google.setOnLoadCallback(updateUIWithVideoContext);
-
 /**
  * Retrieve the context of a Youtube video
+ * @param {String} urlInput inputted url of video
  * @return {Json} Json object of video context information
  */
-async function getVideoContext() {
-  const urlInput = document.getElementById('link-input');
-  const url = extractYouTubeUrl(urlInput.value);
+async function getVideoContext(urlInput) {
+  const url = cleanseUrl(urlInput);
   const response = await fetch('/VideoContext?url=' + url);
   const context = await response.json();
   return context;
 }
 
 /**
  * Fetches data and adds to html
+ * @param {string} url of the video being analyzed
  */
-async function updateUIWithVideoContext() {
-  $('form').submit(async function() {
-    videoContext = await getVideoContext();
-    document.getElementById('video-context').innerHTML =
+async function updateUIWithVideoContext(url) {
+  clearElement('video-context');
+  if (url == undefined) {
+    url = document.getElementById('link-input').value;
+  }
+  videoContext = await getVideoContext(url);
+  document.getElementById('video-context').innerHTML =
       videoInfoAsHTML(videoContext);
```

Comment:
I got very confused when I saw videoContext and videoInfo, and took me a second to realize they are actually the same thing.
I know we have a class called VideoInformation already, but it would have been better if the names are consistent:ContextServlet.java with VideoContext.java, or InformationServlet.java with VideoInformation.java.
Maybe we could just update the names in the js files first to make them consistent?